src/iuse_actor.cpp

```diff
@@ -361,6 +361,7 @@ void consume_drug_iuse::load( JsonObject &obj )
     }
     obj.read( "stat_adjustments", stat_adjustments );
     obj.read( "fields_produced", fields_produced );
+    obj.read( "moves", moves);
```

Comment:
Thanks for help, is the implementation in new commit correct? (I want to keep it optional for json entries).